rpcs3/Emu/RSX/gcm_enums.h

```diff
@@ -102,6 +157,8 @@ namespace gcm
 		CELL_GCM_TEXTURE_D1R5G5B5 = 0x9D,
 		CELL_GCM_TEXTURE_D8R8G8B8 = 0x9E,
 		CELL_GCM_TEXTURE_Y16_X16_FLOAT = 0x9F,
+		CELL_GCM_TEXTURE_COMPRESSED_B8R8_G8R8 = 0xAD,
```

Comment:
This (different values)

Reply:
These enums are bitflags. In this case the linear modifier has been ORed, so the new values are wrong.

Reply:
To be extra clear - revert this. All the other values have 0x20 removed as it has nothing to do with the format itself.